core-web/libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.ts

```diff
@@ -42,7 +43,23 @@ export class SuggestionsListItemComponent implements FocusableOption, OnInit {
 
     scrollIntoView() {
         if (!this.isIntoView()) {
-            this.element.nativeElement.scrollIntoView(this.alignToTop());
+            const child = this.element.nativeElement as HTMLElement;
+            const parent = child.parentElement;
+
+            // Get BoundingClientRect of the elements
+            const {
+                top: containerTop,
+                top: containerBot,
+                height: containerHeight
+            } = parent.getBoundingClientRect();
+            const { top, bottom } = child.getBoundingClientRect();
+
+            const scrollTop = top - containerTop;
+            const scrollBot = bottom - containerBot;
+
+            // += scrollTop -> If we're near the top of the list.
+            // += scrollBot - containerHeight -> If we're near the bottom of the list.
+            parent.scrollTop += this.alignToTop() ? scrollTop : scrollBot - containerHeight;
```

Comment:
What is this for? to scroll into view a list item?

Reply:
Yes, it is. We were using `scrollIntoView()` method but it has some issues. Specifically, this [issue](https://stackoverflow.com/questions/58311605/how-to-make-scroll-inside-the-div-with-scrollintoview-without-scrolling-the-whol)